components/eamxx/src/physics/rrtmgp/tests/rrtmgp_unit_tests.cpp

```diff
@@ -841,3 +842,207 @@ TEST_CASE("rrtmgp_aerocom_cloudtop") {
 
   yakl::finalize();
 }
+
+/* comment out for now
```

Comment:
I would pull this block out of the PR. If it isn't ready to include, let's not commit commented out code.